pkg/telemetryquerylanguage/tql/README.md

```diff
@@ -17,15 +17,41 @@ Invocations represent a function call. Invocations are made up of 2 parts
 - a string identifier. The string identifier must start with a letter or an underscore (`_`).
 - zero or more Values (comma separated) surrounded by parentheses (`()`).
 
-**The TQL does not define any functions implementations.** Users must supply a map between string identifiers and the actual function implementation.  The TQL will use this map and reflection to generate Invocations, that can then be invoked by the user.
+**The TQL does not define any function implementations.** Users must supply a map between string identifiers and the actual function implementation.  The TQL will use this map and reflection to generate Invocations, that can then be invoked by the user.
 
 Example Invocations
 - `drop()`
 - `set(field, 1)`
 
+#### Invocation paramaters
+
+The TQL will use reflection to determine parameter types when parsing an invocation within a statement.
+
+The following types are supported for single parameter values:
+- `Setter`
+- `GetSetter`
+- `Getter`
+- `Enum`
+- `string`
+- `float64`
+- `int64`
+- `bool`
+
+For slice parameters, the following types are supported:
+- `string`
+- `float64`
+- `int64`
+- `uint8`. Slices of bytes will be interpreted as a byte array.
+- `Getter`
+
 ### Values
 
-Values are the things that get passed to an Invocation or used in an Expression. Values can be either a Path, a Literal, an Enum, or an Invocation.
+Values are passed as input to an Invocation or are used in an Expression. Values can take the form of:
+- [Paths](#paths).
+- [Literals](#literals).
+- [Enums](#enums).
+- [Slices](#slices).
```

Comment:
This isn't technically correct, since the language cannot differentiate between arguments that are individual versus combined into a slice as part of the function definition.  The rest of the change is good, but I think Slices should be removed.

Reply:
I somehow missed that, I've removed them as a Value type.